TEMPLATE.md

```diff
@@ -1,4 +1,7 @@
-# <Name of Feature>
+# {Name of Feature}
+
+- Steward: {Your Name} {you@example.com}, {@you} on the Elm Slack
+- Questions + Collaboration: {room} on the Elm Slack
```

Comment:
I don't think anyone can create a channel on the Elm Slack. It seems that I can't for exemple (I don't have any "+" sign aside the channels pannel on the left).